modules/get-started/pages/quick-start.adoc

```diff
@@ -149,21 +360,12 @@ Output:
 TOPIC       STATUS
 chat-room  OK
 ```
-+
-[TIP]
-====
-If you deployed three brokers, you can configure your topics with a replication factor of three. This replication factor provides high availability and data durability. For more details, see xref:develop:config-topics.adoc#choose-the-replication-factor[Choose the replication factor].
-
-```bash
-docker exec -it redpanda-0 rpk topic create chat-room --replicas 3
-```
-====
-// (step {"action":"runShell", "command": "rpk topic create chat-room", "workingDirectory": "../test-resources/single-broker"})
+// (step {"action":"runShell", "command": "rpk topic create chat-room --replicas 3 -X user=superuser -X pass=secretpassword"})
 
 . Produce a message to the topic:
 +
 ```bash
-docker exec -it redpanda-0 rpk topic produce chat-room
+docker exec -it redpanda-0 rpk topic produce chat-room -X user=superuser -X pass=secretpassword
 ```
 
 . Enter a message, then press kbd:[Enter]:
```

Comment:
```suggestion
. Enter a message, then click kbd:[Enter]:
```

Reply:
this is a keyboard button, so I think press is more suitable.